Aztec/Classes/GUI/FormatBar/FormatBar.swift

```diff
@@ -443,7 +443,10 @@ open class FormatBar: UIView {
             heightConstraint?.constant = newHeight
             updateDividerInsets()
             invalidateIntrinsicContentSize()
-            layoutIfNeeded()
+
+            DispatchQueue.main.async {
```

Comment:
I trust that this `async` is necessary 😀 

What do you think about adding a comment to explain why it's there?